src/main/resources/db/migration/common/V1_232__Update_TransactionUrgency_Table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE transcription_urgency ADD COLUMN IF NOT EXISTS display_state BOOLEAN NOT NULL DEFAULT TRUE;
+
+INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(4, 'Up to 3 working days');
+INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(5, 'Up to 7 working days');
+INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(6, 'Up to 12 working days');
+
+UPDATE darts.transcription_urgency SET display_state = false WHERE tru_id = 1;
+UPDATE darts.transcription_urgency SET display_state = false WHERE tru_id = 3;
```

Comment:
This could be one statement

```suggestion
UPDATE darts.transcription_urgency
SET display_state = false
WHERE tru_id IN (1, 3);
```

Reply:
Updated